ios/FluentUI.Demo/FluentUI.Demo/Demos/ListVnextDemoController.swift

```diff
@@ -29,6 +29,21 @@ class ListVnextDemoController: DemoController {
             PersonaData(name: "Amanda Brady", subtitle: "Program Manager", avatarImage: UIImage(named: "avatar_amanda_brady"), color: Colors.Palette.magentaPink10.color)
         ]
 
+        var subchildren: [MSFListVnextCellData] = []
+        for index in 0...2 {
+            listCell = MSFListVnextCellData()
+            listCell.title = "Subchild #\(index)"
+            subchildren.append(listCell)
+        }
+
+        var children: [MSFListVnextCellData] = []
+        for index in 0...2 {
+            listCell = MSFListVnextCellData()
+            listCell.title = "Child #\(index)"
+            children.append(listCell)
+        }
+        children[0].children = subchildren
+
         var avatar: AvatarVnext
 
         /// AvatarView section
```

Comment:
How about naming the section "Custom leading view and collapsible sub cells"?

Then you add 2 personas as top level items with a 2 line cell (name on the title and "Tap to show reports" as the subtitle)

Then you can organize them like this (maybe even using more of the personas samples):

- Kat Larrson
  - Ashley McCarthy
  - Allan Munger
    - Amanda Brady
    - Kristin Patterson
    - Persona 1
     
- Persona 2
  - Persona 3
    - Persona 4
    - Persona 5
    - Persona 6
    - Persona 7

And if you implement the section as a button (other comment below), we can probably make the entire section collapsible too (initially appearing expanded).